sinks/types.go

```diff
@@ -19,7 +19,7 @@ import (
 	"fmt"
 )
 
-var argSink = flag.String("sink", "memory", "Backend storage. Options are [memory | influxdb | bigquery]")
+var argSink = flag.String("sink", "memory", "Backend storage. Options are [memory | influxdb | bigquery | gcm ]")
```

Comment:
Today we only support running in GCE (which we check and error out on in New) and we get the auth information from the machine's service accounts (which we also check and error out on).